_posts/2023-10-26-louisvillejefferson-county-metro-government-balance-kentucky/louisvillejefferson-county-metro-government-balance-kentucky.Rmd

```diff
@@ -0,0 +1,135 @@
+
+![Figure 4: This map illustrates the results as follows: pink represents districts that are less competitive, blue represents districts that are more competitive, and yellow represents those falling within the normal range.](images/map_competitiveness.png){fig-align="center" width="80%"}
+
+### Does Reduced Competitiveness Benefit the Incumbent?
+
+The competitiveness of elections in districts can significantly decrease when partisan gerrymandering is employed to reduce the number of districts which could swing towards the rival party. In light of this, I examine whether this reduced competitiveness in Louisville benefits the incumbent Democratic Party (DEM).
+
+In addition to assessing the competitiveness, I calculated the probability of each district being won by either Democrats or Republicans (as depicted in the upper graph of Figure 3). This additional analysis helped me identify potential cases of gerrymandering. For example, if a district is less competitive and primarily Democratic, but the calculated probability suggests a  likelihood of over 55% for a Republican victory, it raises the possibility of gerrymandering.
+
+Figure 3 reveals that out of the 19 districts found to be less competitive, 4 of them (Districts 7, 17, 21, and 26) are currently represented by Democratic candidates, even though the probability indicates a greater likelihood of Republican success, with over 55%. Remarkably, this reduced competitiveness also appears to work in favor of the Republican Party, leading to 2 districts (Districts 19 and 20) that are more favorable to Republicans.
+
+This finding suggests that the reduced competitiveness in Louisville ultimately appears to contribute to creating favorable conditions for the incumbent, resulting in a 2-seat advantage.
+
+## Methods and Materials
+
+### Methods
+
+I sampled 175,000 districting plans for Louisville across two independent runs of the Sequential Monte Carlo (SMC) algorithm. I performed the analysis above with 5,000 samples from one of the SMC runs.
+
+#### Pre-processing Notes
+
+1.   Excluded 32 precincts[^5] for which geographical data were unavailable.
+
+[^5]: The excluded precincts are: A141, A143, A155, B186, E186, E187, G197, H146, H161, H170, H172, H174, I143, J142, J143, J161, J167, K136, M187, M190, M204, M205, M206, M207, M208, M209, N124, N127, N159, O139, Q147, V135.
+
+<!-- -->
+
+2.   Included 5 elections, Mayor, US Senator, County Judge/Executive, County Clerk, and Coroner to calculate the estimated baseline partisanship.
```

Comment:
Could you clarify which years these elections are from? Also, not sure if you copy pasted this directly into the PR comment, but in the PR the numbering skips from 1 to 5 here, so check to make sure the markdown is rendering correctly.